test/test_roslaunch/test/exit_with_code.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Software License Agreement (BSD License)
+#
+# Copyright (c) 2014, The Johns Hopkins University
```

Comment:
The copyright notices look like they were pasted from somewhere else (2014 JHU, 2009 Willow) - do they make sense? I guess here the maintainers need to tell how this should be handled.

Reply:
Looks like I copied it from `test_roslaunch_ros_args.py`.

`Copyright (c) 2008, Willow Garage, Inc` isn't true either, should it be Open Robotics 2021, or my name?